Makefile

```diff
@@ -57,7 +57,9 @@ help:
 	@echo "  make coverage					-- write coverage report to dir htmlcov"
 	@echo "  make lint						-- lint all project files"
 	@echo "  make lint-quick				-- lint all files staged in git"
-	@echo "  make lint-fix					-- fix linting for all js files staged in git"
+	@echo "  make lint-js-fix					-- fix linting for all js files staged in git"
+	@echo "  make lint-html-fix					-- fix linting for all html files in git only structure"
```

Comment:
sorry, maybe a bit dense but what does "in git only structure" mean ?

Reply:
I assume a typo and I was going for files staged in git like above but honestly not sure because that isn't right